src/util/submitBytecodeRequest.ts

```diff
@@ -0,0 +1,9 @@
+export function submitBytecodeRequest(bytecode: string) {
+    return {
+        "clientToolName": "mythxjs-api",
```

Comment:
This is a different clientToolName from submitContractRequest.

The same comment I had their applies here as well - We should have a default value here (and wherever else clientToolName is set), but it should be configurable; as the library will be used by different tools such as truffle-security, sabre, vscode, etc.